src/util/device.rs

```diff
@@ -56,10 +56,6 @@ impl DeviceExt for crate::Device {
         {
             let mut slice = buffer.slice(..).get_mapped_range_mut();
```

Comment:
maybe pass `unpadded_size` here and then use `slice` without brackets?

Reply:
Ah yeah I guess we don't even need `slice` here at all